cypress.config.ts

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
```

Comment:
There's bunch of eslint warnings here

Reply:
Cypress put that stuff there by default. I guess we can always re-add it if we need it!